std_msgs/Header header
string class_name
float32 confidence
geometry_msgs/PointStamped  centroid
#geometry_msgs/Point  TL #Top Left point
#geometry_msgs/Point  TR #Top Right point
#geometry_msgs/Point  BL #Bottom Left point
#geometry_msgs/Point  BR #Bottom Right point
float64 width
float64 length